ivy/functional/backends/torch/linear_algebra.py

```diff
@@ -425,7 +424,25 @@ def svdvals(x: torch.Tensor, /, *, out: Optional[torch.Tensor] = None) -> torch.
 
 # ToDo: re-add int32 support once
 # (https://github.com/pytorch/pytorch/issues/84530) is fixed
-@with_unsupported_dtypes({"2.0.1 and below": ("int32",)}, backend_version)
+@with_unsupported_dtypes(
+    {
+        "2.0.1 and below": (
+            "complex",
+            "bfloat16",
+            "float16",
+            "float64",
+            "int8",
+            "int16",
+            "int32",
+            "int64",
+            "uint8",
+            "uint16",
+            "uint32",
+            "uint64",
+        )
+    },
+    backend_version,
+)
```

Comment:
Again perhaps more simple to add the supported_dtypes decorator with whatever dtypes are supported.